### PR TITLE
Feature/update change log

### PR DESCRIPTION
**Pull request by countcain**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Upgrade iOS SDK to [5.6.0](https://docs.sentiance.com/sdk/changelog/ios#5-6-0-5-feb-2020)
+- Added `reset` method to reset the Sentiance SDK
+
+## [3.0.8] - 2020-01-16
+- Upgrade Android SDK to [4.13.0](https://docs.sentiance.com/sdk/changelog/android#4-13-0-6-jan-2020)
+
+## [3.0.7] - 2019-11-20
+- Upgrade iOS SDK to [5.5.5](https://docs.sentiance.com/sdk/changelog/ios#5-5-5-13-nov-2019)
+- Support the auto-start option when initializing the Sentiance SDK
+- Send `SendUserLink` event during initialization if user linking is enabled (Android Only)
```
Comment (sebouh00) [inline CHANGELOG.md]: The event is `SDKUserLink`, and I believe it's supported for both platforms.

Comment (countcain) [reply CHANGELOG.md]: Updated. Double checked the code, it does exist in iOS as well.

Comment (sebouh00) [reply CHANGELOG.md]: Thanks. Please don't forget to reformat, as I mentioned [here](https://github.com/sentiance/react-native-sentiance/pull/119#pullrequestreview-372577752).

Comment (countcain) [reply CHANGELOG.md]: Updated. :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,12 +6,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Upgrade Android SDK to [4.14.0](https://docs.sentiance.com/sdk/changelog/android#4-14-0-31-jan-2020)
+- Upgrade iOS SDK to [5.6.0](https://docs.sentiance.com/sdk/changelog/ios#5-6-0-5-feb-2020)
+
+## [3.0.8] - 2020-01-16
+### Changed
+- Upgrade Android SDK to [4.13.0](https://docs.sentiance.com/sdk/changelog/android#4-13-0-6-jan-2020)
+
+## [3.0.7] - 2019-11-20
+### Added
+- The ability to update SDK foreground notification
```
Comment (sebouh00) [inline CHANGELOG.md]: Minor remark: This is only applicable to Android.

**Pull request merged**